start-txgh.sh

```diff
@@ -19,4 +19,4 @@ if [[ -n ${NGINX_CHANGES} ]]; then
 fi
 
 tee_logs "Reloading nginx"
-nginx -s reload
+docker exec ec2-user_nginx_1 /bin/bash -c "nginx -s reload"
```

Comment:
It doesn't really matter either way, but you can call `nginx` directly with `docker exec` instead of going through `/bin/bash`:
```
docker exec ec2-user_nginx_1 nginx -s reload
```